code/answering_machine_detection/README.md

```diff
@@ -0,0 +1,96 @@
+# Answering Machine Detection
+This guide utilizes Answering Machine Detection to determine whether a human or voicemail machine has picked up the phone so that it can either dial a number to connect someone to the human or leave a message for the voicemail box. 
+
+When a call is initiated with ```machine_detection``` set to ```DetectMessageEnd```, there is a parameter posted to the webhook called AnsweredBy. The potential options here are machine_end_other, machine_end_beep, machine_end_silence, and human. If the parameter returned is anything other than human, this example will take a short pause to wait for the beep and then play a message for the intended recipient. If the parameter returned is human, it will dial a number to connect the human to another person.
```

Comment:
Use single backticks to mark "keywords", and please mark all of them including the possible return values.